libraries/provider_runit_service.rb

```diff
@@ -136,6 +140,15 @@ def configure_service
             else
               Chef::Log.debug("Control signals not specified for #{new_resource.service_name}, continuing")
             end
+
+            # Create/Delete service down file
+            # To prevent unexpected behavior, require users to explicitly set
+            # delete_downfile to remove any down file that may already exist
+            if new_resource.start_down
+              do_action(downfile, :create)
+            elsif new_resource.delete_downfile
```

Comment:
The idea here is to prevent anyone who has manually created a downfile (because the cookbook as of yet doesn't manage it, or at least no longer manages it) from suddenly having their downfile disappear out from under them.  Having a downfile doesn't keep a service from starting, just keeps it from being started automatically.